### ISSUE TITLE
When using Samsung Dex, a black band appears at the top of the screen and fps drops 30%.

### ISSUE THREAD

**Issue by Raistlin43** [yuzu-emu/yuzu#12546]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

177-178

### Description of Issue

I have always played with my Samsung Galaxy S23 Plus with Samsung Dex on my TV with no problems at full screen and full speed. The problem is that now you see a black strip at the top of the screen, so the game does not look full screen and also when connecting Samsung Dex the fps drops by 30 fps (something that happened to me before and also does not happen without Samsung Dex).
![1704050838839](https://github.com/yuzu-emu/yuzu/assets/136481247/97a1501b-ad16-40ae-841b-7283ca8ea642)

### Expected Behavior

Being able to play with Samsung Dex in full screen and at the same speed as without Samsung Dex, as I have always done. 

### Reproduction Steps

When you connect the Samsung mobile to the TV, Samsung Dex is activated and when I open a game it does not appear in full screen, I get a black strip at the top. In addition the fps drop drastically in all games from 30 fps stable to 23 fps fixed (it is as if I had a limitation when connecting Samsung Dex). 

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13804296/yuzu_log.txt)

### System Configuration

CPU: 8 gen 2 ( Samsung Galaxy S23 Plus) 
GPU/Driver:Adreno 740 Turnip 14
RAM: 8 gb
OS: Android 14

**Comment by goldenx86** [comment]
It seems the Android 14 update introduced this. Notice how if you scroll the mouse to the top, the top bar has exactly the same height as that black bar.

The framerate is most likely the TV setting a 30 Hz refresh rate, check the settings with either the phone or the TV, set it to 60 Hz.

**Comment by Raistlin43** [comment]
> Parece que la actualización de Android 14 introdujo esto. Observe cómo si desplaza el mouse hacia la parte superior, la barra superior tiene exactamente la misma altura que la barra negra.
> 
> Lo más probable es que la velocidad de fotogramas sea que el televisor configure una frecuencia de actualización de 30 Hz; verifique la configuración con el teléfono o el televisor y configúrelo en 60 Hz.

Yes, when you hover the mouse cursor over it, it moves together with the mouse. The funny thing is that the only app that happens to me is Yuzu with Citra for example, since it keeps the full screen and no stripes are seen.

Regarding the speed limitation I don't know why, all my emulators still work at full speed, my TV is 11 years old and lacks those Hz functions and Yuzu always worked fine in previous versions at 30 fps (or 60 fps in the games that supported it). It is worth noting that out of 10 times I tried it, 1 of them was full screen and that time it was at full speed 30 fps, so maybe that black strip still influences the performance of the emulator. Thanks!!

**Comment by RodrigoQuesadaDev** [comment]
Hello, I'm also experiencing the same issue, although in my case, I also get black bars on the sides unless I use the 'Stretch to window' option (which still doesn't fix the top black bar issue).

**Comment by Raistlin43** [comment]
> Hola, yo también estoy experimentando el mismo problema, aunque en mi caso también aparecen barras negras en los lados a menos que use la opción "Estirar a la ventana" (que todavía no soluciona el problema de la barra negra superior).

And FPS? also fall. The stripes on the side are removed in my case by stretching the screen, but the annoying top black stripe still appears... this did not happen in previous versions of Yuzu...
